app/models/manageiq/providers/workflows/automation_manager/workflow_instance.rb

```diff
@@ -45,8 +45,8 @@ def queue_callback(state, message, _result)
       miq_task.update_status(MiqTask::STATE_ACTIVE, MiqTask::STATUS_OK, "Workflow running") # TODO: Can we get the last state here?
     when "success"
       miq_task.update_status(MiqTask::STATE_FINISHED, MiqTask::STATUS_OK, "Workflow completed successfully")
-    when "error"
-      miq_task.update_status(MiqTask::STATE_FINISHED, MiqTask::STATUS_ERROR, "Workflow completed in failure") # TODO: Not sure if this should be MiqTask::STATUS_WARN instead?
+    when "failure"
```

Comment:
Workflow.status is "failure" not "error" per https://github.com/ManageIQ/floe/blob/master/lib/floe/workflow/context.rb#L71